### PR TITLE
Fix runners attributes with resource pack script

### PR DESCRIPTION
**Pull request by heurtematte**

### CODE REVIEW

**Comment by heurtematte** [comment]
NOTE: not perfect but I have added a FIXME comment when the script add a new resourcepack attribut to config.jsonnet but displayname is not ended by a coma. 

I tried with jsonnet cmd line but for config.jsonnet that includes custom template `currentCloud.templates["centos-8"], compilation fails.
We need to find a clean solution for template inclusion in projects config files.

File: jenkins/add_resource_pack.sh [hunk 1]
```diff
@@ -48,6 +49,16 @@ if [[ "${DEDICATED_AGENTS}" -lt 0 ]]; then
   exit 1
 fi
 
+if [ -z "${RUNNERS}" ]; then
+  printf "ERROR: Number of dedicated runners must be given.\n"
```
Comment (fredg02) [inline jenkins/add_resource_pack.sh:53]: Please replace "dedicated runners" with "GitHub large runners" everywhere.

File: jenkins/add_resource_pack.sh [hunk 2]
```diff
@@ -140,6 +153,7 @@ update_jiro_config() {
     new_value="$((1 + RESOURCE_PACKS))"
     new_line="   resourcePacks: ${new_value},"
     #TODO: simplify?
+    #FIXME does not work if diplayName is not ended with a comma
```
Comment (fredg02) [inline jenkins/add_resource_pack.sh:156]: This should work:
```
#Adds missing comma if required
if [[ $(grep "displayName" "${config_file}" | grep -o '.$') != "," ]]; then
  sed -i -r "s/(displayName.*$)/\1,/g" "${config_file}"
fi
```

**Review by fredg02 — CHANGES_REQUESTED** [review]
See my comment above.

**Comment by heurtematte** [comment]
done!

**Review by fredg02 — APPROVED** [review]

**Pull request merged**